### ISSUE TITLE
0.1.1 "Package Resolution Failed"

### ISSUE THREAD

**Issue by sweeneybros** [music-notation-swift/music-notation-core#152]
I'm sorry to post this very beginner question, but I am having trouble adding this package to my Swift project using the "Add Package Dependency" function in Xcode. When I enter the URL for the repository, it seems to recognize is it just fine and offer the option to include the 0.1.1 version, but then it throws this error
![Screen Shot 2020-10-18 at 3 08 24 PM](https://user-images.githubusercontent.com/58794248/96386977-df39d880-1153-11eb-8bc5-e74234f10ed2.png)
Can anyone offer any tips or resources on how to resolve this issue?

**Comment by woolie** [comment]
Can you tell me what version of Xcode, and what platforms you are trying to build for?

**Comment by sweeneybros** [comment]
I'm running Xcode 11.7 (11E801a) on macOS 10.15.7 

Building for iOS

**Comment by woolie** [comment]
I will try and fix this soon. SPM use of the library wasn't really planned out and so I will do so now and push it when finished.

**Comment by sweeneybros** [comment]
wow, thank you so much!

**Comment by sweeneybros** [comment]
I didn't ask, but is there an alternate way to include this library in my iOS app? Or if that's basic know-how, can you point me to a resource that describes how to do it?

**Comment by woolie** [comment]
Yes. Turning this into an SPM package has turned out to be more work than I anticipated. I'll put some instructions for manually adding the library in the README.

**Comment by woolie** [comment]
> I didn't ask, but is there an alternate way to include this library in my iOS app? Or if that's basic know-how, can you point me to a resource that describes how to do it?

OK. I added instructions for the simplest way to use the project as a dependency. NB; I added the steps for the macOS version of the library, but it is the same steps for iOS except picking the iOS bits.

HTH

**Comment by sweeneybros** [comment]
Thank you so much. I'll take a look at the instructions
